cddl/contrib/opensolaris/tools/ctf/cvt/hash.c

```diff
@@ -87,7 +87,7 @@ hash_name(int nbuckets, const char *name)
 {
 	const char *c;
 	ulong_t g;
-	int h = 0;
+	unsigned int h = 0;
```

Comment:
ulong_t might make more sense given g, that or make g unsigned int given bits 32 and above are never used... (in fact only bits 24 through 31)

Reply:
`ulong_t` sounds good to me.